packages/vue-app/template/router.js

```diff
@@ -97,5 +97,9 @@ export function createRouter() {
     <% if (router.parseQuery) { %>parseQuery: <%= serializeFunction(router.parseQuery) %>,<% } %>
     <% if (router.stringifyQuery) { %>stringifyQuery: <%= serializeFunction(router.stringifyQuery) %>,<% } %>
     fallback: <%= router.fallback %>
-  })
+  }
+}
+
+export function createRouter() {
```

Comment:
Should we also allow passing `routerOptions` as an argument and default it to `routerOptions()` ? PS Did you validated this change against router-module? Can you please make a preview PR?

Reply:
I actually wanted to use the router options and create my custom router.

Reply:
I will pass routerOptions as parameter